xbmc/video/test/TestVideoFileItemClassify.cpp

```diff
@@ -113,9 +113,9 @@ TEST_P(VideoTest, IsVideo)
 const auto video_tests = std::array{
     VideoClassifyTest{"/home/user/video.avi", true, "video/avi"},
     VideoClassifyTest{"/home/user/video.avi", true, "", 1},
-    VideoClassifyTest{"/home/user/video.avi", false, "", 2},
-    VideoClassifyTest{"/home/user/video.avi", false, "", 3},
-    VideoClassifyTest{"/home/user/video.avi", false, "", 4},
+    VideoClassifyTest{"/home/user/video.gam", false, "", 2},
+    VideoClassifyTest{"/home/user/video.mus", false, "", 3},
+    VideoClassifyTest{"/home/user/video.pic", false, "", 4},
```

Comment:
I suggest to keep the existing checks and fix them and to add the new ones in addition. They check different things so it's valuable to have them both.

Reply:
no need to have 3 duplicates of an already existing test? There is already a test that checks for mime type override. These tests are specifically there to test that tags make it return false.